accepted/PSR-12-extended-coding-style-guide.md

```diff
@@ -191,7 +193,8 @@ use Vendor\Package\SomeNamespace\{
 };
 ~~~
 
-And the following would not be allowed:
+And the following would *not* be allowed:
```

Comment:
Maybe here "SHOULD/MUST NOT" might be more suited?

Reply:
I just fixed a few doubts I had, the PSR is fresh, not really intending to modify the M/S/C/W.

But maybe we should open another discussion regarding this one, IDK.

Reply:
Not sure what `M/S/C/W` means... 

If it means `must`/`should`/`could`/`would` then according to the first paragraph of the standard there is no `could` or `would`.. for everyone to be on the same page :)

> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.

Never the less I do agree that this particular part should be on a separate discussion.
Will try to open a new PR related to this... 

I just happened to notice it now that you applied these changes... Will leave this open until the new PR.

Reply:
Sure. :)

Thanks for the note, though!